public/js/components/Editor.css

```diff
@@ -42,10 +41,6 @@
   background-color: var(--theme-body-background);
 }
 
-.CodeMirror-linenumber {
-  font-size: 11px;
-}
```

Comment:
ooh, i think we need this for chrome actually. for some reason chrome defaults to 12px and firefox defaults to 11.